env.sample

```diff
@@ -0,0 +1,13 @@
+export API_HOST=localhost
+export PORT=2015
+export POSTGRE_CONNECTION_STRING=postgresql://localhost:5432/skittles
+
+## log level - given a chosen level, log events only at or above that level
+## debug < info < warn < error < exception < stat
+export LOG_LEVEL=info
+
+## Log to file - LOG_FILE_PATH and LOG_FILENAME must be defined
```

Comment:
I have a gut feeling we won't want to use this, and we'll only want to log to stdout, but who knows...